docs/content/guides/dagster/managing-ml.mdx

```diff
@@ -0,0 +1,271 @@
+    auto_materialize_policy=AutoMaterializePolicy.lazy(),
+    freshness_policy=FreshnessPolicy(maximum_lag_minutes=7 * 24 * 60),
+)
+def my_other_ml_model(my_other_data):
+    ...
+```
+
+This can be useful if you know that you want your machine learning model retrained at least once a week. While Dagster allows you to refresh a machine learning model as often as you like, best practice is to re-train as seldomly as possible. Model retraining can be costly to compute and having a minimal number of model versions can reduce the complexity of reproducing results at a later point in time.
+
+The other way we can use lazy auto-materializations is without a `freshness_policy`. This will trigger the auto-materialization only if a downstream asset requires `my_ml_model`.
+
+This can be useful if you want your machine learning model to be refreshed whenever the model is being used for a prediction. In this case, we are using the model once a week for `predictions`, ensuring that `my_ml_model` is retrained before it is used.
+
+```python file=/guides/dagster/managing_ml/managing_ml_code.py startafter=without_policy_start endbefore=without_policy_end
+from dagster import AutoMaterializePolicy, FreshnessPolicy, asset
+
+
+@asset
+def some_data():
+    ...
+
+
+@asset(auto_materialize_policy=AutoMaterializePolicy.lazy())
+def some_ml_model(some_data):
+    ...
+
+
+@asset(
+    auto_materialize_policy=AutoMaterializePolicy.lazy(),
+    freshness_policy=FreshnessPolicy(maximum_lag_minutes=7 * 24 * 60),
+)
+def predictions(some_ml_model):
+    ...
+```
+
+We can also use a more traditional schedule on our machine learning assets, causing them to be re-materialized or retrained on the latest data. For example, setting up a [cron schedule on a daily basis](/concepts/partitions-schedules-sensors/schedules).
+
+This can be useful if you have data that is also being scheduled on a cron schedule and want to add your machine model jobs to run on a schedule as well.
+
+```python file=/guides/dagster/managing_ml/managing_ml_code.py startafter=basic_schedule_start endbefore=basic_schedule_end
+from dagster import AssetSelection, define_asset_job, ScheduleDefinition
+
+ml_asset_job = define_asset_job("ml_asset_job", AssetSelection.groups("ml_asset_group"))
+
+basic_schedule = ScheduleDefinition(job=ml_asset_job, cron_schedule="0 9 * * *")
+```
+
+### Monitoring
+
+Integrating your machine learning models into Dagster allows you to see when the model and its data dependencies were refreshed, or when a refresh process has failed. By using Dagster to monitor performance changes and process failures on your ML model, it becomes possible to set up remediation paths, such as automated model retraining, that can help resolve issues like model drift.
+
+In this example, the model is being evaluated against the previous model’s accuracy. If the model’s accuracy has improved, the model is returned for use in downstream steps, such as inference or deploying to production.
```

Comment:
This seems like a fairly advanced use case to start with.  I would expect something more basic where we yield metadata alongside our model and show how we can track it in the UI.

When we do include this model, it should probably get its own subheading like "Conditionally overwriting a model based on accuracy" or something.

Reply:
Does the section below on metadata cover it?